Dockerfile

```diff
@@ -18,7 +18,12 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
+# Use .env for environment variables
+ARG ENV_FILE_PATH=.env
+ENV ENV_FILE=${ENV_FILE_PATH}
```

Comment:
Using an ARG for the .env file path is a good start, but ensure that sensitive information is not hardcoded. Consider using Docker secrets or environment variables passed at runtime for better security.